extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-service.js

```diff
@@ -45,6 +45,20 @@ describes.realWin('amp-recaptcha-service', {
         });
   });
 
+  it('should store the iframe src origin ' +
+    'to be used in message origin checking', () => {
+    expect(recaptchaService.registeredElementCount_).to.be.equal(0);
+    return recaptchaService
+        .register(fakeSitekey).then(() => {
+          expect(recaptchaService.registeredElementCount_).to.be.equal(1);
+          expect(recaptchaService.iframe_).to.be.ok;
+
+          expect(recaptchaService.recaptchaFrameOrigin_).to.be.ok;
+          expect(recaptchaService.recaptchaFrameOrigin_
```

Comment:
this test does not seem to be very useful as it's making too much assumption of the implementation details.

I'd rather mock out `postMessage` (external dependency) and check the param get passed in.